rmf_fleet_adapter_python/src/adapter.cpp

```diff
@@ -126,6 +126,8 @@ PYBIND11_MODULE(rmf_adapter, m) {
     py::arg("battery_soc"),
     py::call_guard<py::scoped_ostream_redirect,
     py::scoped_estream_redirect>())
+  .def("override_status", &agv::RobotUpdateHandle::override_status,
+    py::arg("battery_soc"))
```

Comment:
Is ```py::arg("battery_soc"))``` correct?

Reply:
Nope, that's definitely a copy/paste error. I'll correct that in an upcoming PR.